Sources/ProjectDescription/DeploymentDevice.swift

```diff
@@ -10,6 +10,8 @@ public struct DeploymentDevice: OptionSet, Codable, Hashable {
     public static let ipad = DeploymentDevice(rawValue: 1 << 1)
     /// A Mac device.
     public static let mac = DeploymentDevice(rawValue: 1 << 2)
+    /// An Apple Vision device.
+    public static let vision = DeploymentDevice(rawValue: iphone.rawValue | ipad.rawValue | mac.rawValue)
```

Comment:
this is not needed,right?
```suggestion
```

Reply:
There needs to be a way to express an ipad app operating in compatibility mode for an ios target

Reply:
<img width="317" alt="image" src="https://github.com/tuist/tuist/assets/6877780/9e64a40d-69f8-4127-8af1-a23defb0202f">